UPGRADING.md

```diff
@@ -67,8 +68,8 @@ that we plan to make to this set of commands and options.
   This is the only bundler command requiring external dependencies, both an OS
   dependency (the `graphviz` package) and a gem dependency (the `ruby-graphviz`
   gem). Removing these dependencies will make development easier and it was also
-  seen by the bundler team as an opportunity to develop a [bundler
-  plugin](https://github.com/bundler/bundle-viz) that it's officially maintained
+  seen by the bundler team as an opportunity to develop a bundler
+  plugin that it's officially maintained
   by the bundler team, and that users can take as a reference to develop their
   own plugins. The plugin contains the same code as the old core command, the
```

Comment:
We should probably replace "contains" with "will contain", and remove the finaly sentence in this paragraph about checking the plugin's README.

Reply:
Okay. I also added a link to https://github.com/rubygems/bundler/issues/7041 so that readers can get the latest updates.